src/Helpers.elm

```diff
@@ -0,0 +1,11 @@
+module Helpers exposing (onChange)
+
+import Html exposing (..)
```

Comment:
Do we need to import both `Html` and `Html.Attribute`? I assume one of the provides `Attribute`